### PR TITLE
Fix `hide-low-quality-comments` style

### LINKED ISSUE #4849: `hide-low-quality-comments` doesn't full colorize the border
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Description

Notice the right side of the avatar:

image

### Screenshot

_No response_

### Console errors

_No response_

### Example URL

https://github.com/jerone/UserScripts/issues/1#issuecomment-162996100

Note: the comment is hidden, you need to show it at the bottom of the page.

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by kidonng**
Fix #4849

NOTE: `highest-rated-comment` got a similar style fix in #4554

## Test URLs

https://github.com/jerone/UserScripts/issues/1#issuecomment-162996100

## Screenshot

Before: image

After: 
image

### CODE REVIEW

**Comment by fregante** [comment]
Hey I implemented this bug 3 years ago or something 😌 I'm proud it almost made it to 2022

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
Might be a good chance to update that red to use a dark-mode-compatible CSS variable

**Comment by fregante** [comment]
Thanks! We can drop the media query for the 3-color fallback. They're only there to avoid defaulting to transparent, it's fine if they're not perfect in dark mode when missing. They'll actually be useful to show us the variables aren't working.

**Comment by kidonng** [comment]
> They'll actually be useful to show us the variables aren't working. 

It's better we just have a debug mode where it automatically scans for all CSS variables and report missing ones.

File: source/options.css [hunk 1]
```diff
@@ -12,7 +12,13 @@
 }
 
 :root {
-	--github-red: #cb2431;
+	--github-red: #cf222e;
+}
+
+@media (prefers-color-scheme: dark) {
+	:root {
+		--github-red: #f85149;
```
Comment (kidonng) [inline source/options.css:20]: This can stay I suppose

**Comment by fregante** [comment]
> > They'll actually be useful to show us the variables aren't working.
> 
> It's better we just have a debug mode where it automatically scans for all CSS variables and report missing ones.

I've been thinking about setting up automated tests running on GitHub.com with a real user. But it takes a lot of effort to set up and a lot of effort to write tests. I'd rather focus on the bug fixing than writing something that just finds more bugs 😜

**Pull request merged**